src/cljs/webchange/editor_v2/dialog/dialog_text_form/side_menu/main/voice_over/current_audio/views.cljs

```diff
@@ -16,6 +16,14 @@
     " audio or "
     [:span.clickable "select from available"]]])
 
+(defn- settings
+  []
+  (let [handle-click #(re-frame/dispatch [::state/open-voice-over-audio-window])]
+    [:div.settings
+     [icon-button {:icon     "settings"
```

Comment:
I think [this icon](https://fonts.google.com/icons?selected=Material+Icons&icon.query=+Open+In+Full) would be more suitable